### ISSUE TITLE
Is there way to force i3 work with yabar as dock?

### ISSUE THREAD

**Issue by neg-serg** [geommer/yabar#129]
I've tried both default i3 config and mine.

When I've tried to start yabar with following config: http://sprunge.us/JUYC
(or any another) yabar did not pulled into bar placeholder.

While dzen2, for example, does. I've tried to create a placeholder with dzen2 and  make it fully transparent, but for some reason my i3 does not respect the following snippet:

```
exec dzen2 -y -1 -h 38 -dock & sleep 0.5s && transset-df --id $(printf '0x%x' $(xdotool search --classname dzen2)) 0
exec yabar -c ~/.config/yabar/main.conf 
```

if it added in ~/.config/i3/config

Also I've created question at https://www.reddit.com/r/i3wm/comments/5v9dlk/bar_placeholders/ no success

Is that true that yabar developed with i3 reliability in mind?

Any ideas?

Also it looks like yabar did not respect ewmh/i3 hints at all, because of when you get window fullscreen yabar stay there and didn't hide.

**Comment by NBonaparte** [comment]
Hi, could you provide some screenshots? Also, try using `xprop` and check if `_NET_WM_WINDOW_TYPE(ATOM) = _NET_WM_WINDOW_TYPE_DOCK`, and the strut values are correct.

I'll try to fix the window stacking on i3 soon.

**Comment by NBonaparte** [comment]
I believe this has been fixed since 63f0bb2, please close if it is.

**Comment by neg-serg** [comment]
Yes, it works well for now, thank you.
Sorry for that, I was working a bit...
